pkgs/by-name/uf/ufetch/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  stdenvNoCC,
+  fetchFromGitLab,
+  lib,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ufetch";
+  version = "0.3";
+
+  src = fetchFromGitLab {
+    owner = "jschx";
+    repo = "ufetch";
+    rev = "v${version}";
+    hash = "sha256-1LtVCJrkdI2AUdF5d/OBCoSqjlbZI810cxtcuOs/YWs=";
+  };
+
+  patches = [ ./0001-optimize-packages-command.patch ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 ufetch-nixos $out/bin/ufetch
```

Comment:
I think it would be good to install the other variants too. nixpkgs can also be used on other distros.